descqa/EllipticityDistribution.py

```diff
@@ -326,11 +344,29 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                 if self.first_pass: #add validation data if evaluating first catalog
                     self.validation_subplot(summary_ax_this, self.validation_data.get(morphology), validation_label)
                 self.decorate_subplot(summary_ax_this, n, label=cutlabel)
+                
+                #check ellipticity distributions
+                number_passed, percentiles = self.validate_percentiles(N)
+                print("Percentiles for morphology {} are: ".format(morphology)+', '.join([" {:.3f} ({})".format(p, v) for p,v in zip(percentiles, self.validation_percentiles['percentiles'])]))
+                if number_passed>0:
+                    print("Ellipticity percentile check failed for morphology {}".format(morphology))
+                n_fails += number_passed
+                
 
             else:
                 #make empty subplots invisible
                 ax_this.set_visible(False)
                 summary_ax_this.set_visible(False)
+        
+        #check overall ellipticity distribution
+        global_N = np.sum(np.sum(N_array, axis=1), axis=0)
+        number_passed, percentiles = self.validate_percentiles(global_N)
+        print("Percentiles for global distribution are:".format(morphology)+', '.join([" {:.3f} ({})".format(p, v) for p,v in zip(percentiles, self.validation_percentiles['percentiles'])]))
```

Comment:
the first `.format` is not actually being used.

Reply:
Okay, I think this is fixed now...